### DISCUSSION TITLE
Identify Cells Used In Differential Testing

### DISCUSSION THREAD

**Discussion by armonazizi** [GreenleafLab/ArchR#811] [Questions / Documentation] [answered]
Hello Jeff and Ryan,

Thanks for this great tool! I'm using `getMarkerFeatures` to perform differential testing of peak accessibility between 2 cell clusters. Is there a way to identify or have the function return the cells that were used in doing the differential test? In other words, is there a straightforward way to obtain the cells that were returned from the internal `matchBiasCellGroups` function?

I'd like to obtain the bias-matched cell group IDs so that I can perform some downstream analysis and plotting that will match the output from `getMarkerFeatures`.

Thank you!!

-Armon

**Comment by rcorces** [comment, marked as answer]
I believe this info is stored in the `SummarizedExperiment` object under `metadata(se)$MatchInfo`
https://github.com/GreenleafLab/ArchR/blob/968e4421ce7187a8ac7ea1cf6077412126876d5f/R/MarkerFeatures.R#L297

**Reply by armonazizi** [reply]
Thanks for the quick reply!
